infra/ansible/roles/knot_authoritative/templates/knot.conf.j2

```diff
@@ -26,11 +26,31 @@ template:
     storage: /var/lib/knot/zones
 
 key:
+  - id: grandmox.prox.mesh.nycmesh.net.
+    algorithm: hmac-sha512
+    secret: {{ TSIG_KEY_GRANDMOX }}
+  - id: jon.prox.mesh.nycmesh.net.
+    algorithm: hmac-sha512
+    secret: {{ TSIG_KEY_JON }}
+  - id: nycmesh-10-r630-01.prox.mesh.nycmesh.net.
+    algorithm: hmac-sha512
+    secret: {{ TSIG_KEY_10_R630_01 }}
+  - id: nycmesh-713-r640-01.prox.mesh.nycmesh.net.
+    algorithm: hmac-sha512
+    secret: {{ TSIG_KEY_713_R640_01 }}
   - id: doh.mesh.nycmesh.net.
     algorithm: hmac-sha512
     secret: {{ TSIG_KEY_DOH }}
 
 acl:
+{% for subdomain in "grandmox;jon;nycmesh-10-r630-01;nycmesh-713-r640-01".split(';') %}
```

Comment:
Could this somehow be multiline to improve readability, or maybe have a variable declared somewhere?

Reply:
I went back and forth on this, would it be better to spell them out instead of the loop (as above)?

Reply:
Unrolled